ext/DynamicQuantitiesLinearAlgebraExt.jl

```diff
@@ -1,8 +1,69 @@
 module DynamicQuantitiesLinearAlgebraExt
 
-import LinearAlgebra: norm
-import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity
+import LinearAlgebra: norm, inv, (\), svd, Algorithm, default_svd_alg, SVD, Diagonal, Adjoint, Transpose, AbstractRotation, AbstractMatrix, eigen, eigsortby, Eigen, det
+import DynamicQuantities: AbstractQuantity, ustrip, dimension, new_quantity, AbstractDimensions, QuantityArray, Quantity
+
+const QuantityArrayVecOrMat{T} = Union{QuantityArray{T,2},QuantityArray{T,1}} where T
 
 norm(q::AbstractQuantity, p::Real=2) = new_quantity(typeof(q), norm(ustrip(q), p), dimension(q))
 
+\(q::QuantityArrayVecOrMat,r::QuantityArrayVecOrMat) = QuantityArray(ustrip(q)\ustrip(r),dimension(r)/dimension(q))
```

Comment:
Small nitpick and I'm happy to do it but could you space each method definition out? i.e., 
```suggestion
\(q::QuantityArrayVecOrMat, r::QuantityArrayVecOrMat) = QuantityArray(ustrip(q) \ ustrip(r), dimension(r) / dimension(q))
```